src/DurableEngine/Models/OrchestrationContext.cs

```diff
@@ -23,7 +23,9 @@ public class OrchestrationContext
         /// Input to the orchestrator.
         /// </summary>
         [DataMember]
-        public object Input { get; set; }
+        public object Input { 
```

Comment:
Nit: please add a newline before the opening braces here so we're consistent with the style of the rest of the codebase